package.json

```diff
@@ -69,9 +68,11 @@
     "build": "npm run config:build && npm run js:build && npm run styles:build && npm run templates:build",
     "watch": "npm run dev && npm-run-all -p js:watch styles:watch templates:watch config:watch",
     "js:ci": "eslint app/\\*\\*/\\*.js",
+    "js": "npm run js:dev",
     "js:dev": "gulp js:dev",
     "js:build": "gulp js:build",
     "js:watch": "gulp js:watch",
+    "styles": "npm run styles:dev",
```

Comment:
We have something similar on the WP theme where we have a `styles` task and a `js` task that is an alias for build the dev assets.